packages/core/src/types/Colormap.ts

```diff
@@ -15,7 +15,8 @@ type OpacityMapping = {
 
 type ColormapPublic = {
   /** name of the colormap */
-  name: string;
+  name?: string;
+  opacity?: number;
```

Comment:
I prefer this one and opacityMapping to be the same thing. So we can call them opacity (remove the opacityMapping), and in the code we check whether it is a number of array of objects and with documentation explain what each one do. How about that